rolling-shutter/medley/db/sqlc.go

```diff
@@ -48,32 +48,47 @@ func ParseSQLC(filesystem fs.FS, sqlcPath string) ([]Schema, error) {
 
 	schemas := []Schema{}
 	for _, entry := range d.SQL {
-		// TODO: allow for individual sql files and multiple dirs.
-		// For now assume we only get one string
-		// which is the path of the directory containing the schema files.
-		// This won't handle multiple directories, or individual sql file names.
-		schemaDirPath := path.Join(sqlcDir, entry.Schema)
-		dirEntries, err := fs.ReadDir(filesystem, schemaDirPath)
-		if err != nil {
-			return nil, errors.Wrap(err, "error reading schema dir")
+		var schemaPaths []string
+
+		// This allows only for schema directories, not individual sql files.
+		switch schema := entry.Schema.(type) {
+		case string:
+			schemaPaths = []string{schema}
+		case []interface{}:
+			for _, s := range schema {
+				if str, ok := s.(string); ok {
+					if str == "migrations" {
```

Comment:
can you add a comment explaining why the migrations dir is skipped? I think this might become hard to understand in the future.